setup.cfg

```diff
@@ -69,7 +69,7 @@ install_requires =
     jinja2==3.0.*
     metomi-isodatetime==1!3.0.*
     # Constrain protobuf version for compatible Scheduler-UIS comms across hosts
-    protobuf==3.19.*
+    protobuf==4.21.*
```

Comment:
Should this be `>=4.21.2,<4.22.0` too? Or is the issue with `4.21.0`, `4.21.1` just with the conda-forge packages?

(if so worth sticking a comment in here so we don't get confused when updating the conda-forge feedstock at release time).